moto/dynamodb2/models.py

```diff
@@ -110,27 +111,27 @@ def describe_attrs(self, attributes):
         }
 
     def update(self, update_expression, expression_attribute_names, expression_attribute_values):
-        ACTION_VALUES = ['SET', 'set', 'REMOVE', 'remove']
-
-        action = None
-        for value in update_expression.split():
-            if value in ACTION_VALUES:
-                # An action
-                action = value
-                continue
-            else:
+        parts = [p for p in re.split(r'\b(SET|REMOVE|ADD|DELETE)\b', update_expression) if p]
```

Comment:
As far as I understand the DynamoDB docs (I haven't used this feature myself) it is legal to include multiple update statements in one update expressions as long as they don't use the same operator. Using `match` in this straightforward manner would include any following ones in the `attribute_pairs` group. We could design a fancier regex and use `findall`, but I believe the only way expressions are identified is by occurrence of one of the operator keywords, so I doubt that this expression would be more readable.